src/tools/clippy/src/driver.rs

```diff
@@ -166,6 +166,8 @@ impl rustc_driver::Callbacks for ClippyCallbacks {
         // MIR passes can be enabled / disabled separately, we should figure out, what passes to
         // use for Clippy.
         config.opts.unstable_opts.mir_opt_level = Some(0);
+        config.opts.unstable_opts.mir_enable_passes =
+            vec![("CheckNull".to_owned(), false)];
```

Comment:
Should we also add CheckAlignment here with exactly the same reasoning?

Reply:
Fair point! I added it :)